### ISSUE TITLE
Insert picture not working when using SummernoteInplaceWidget

### ISSUE THREAD

**Issue by tomas-nz** [summernote/django-summernote#414]
Using insert picture from Summernote toolbar is working fine for me in iframe widget (SummernoteWidget). However I am unable to make it work when I use the in-place widget (SummernoteInplaceWidget). When I select an image via the GUI function nothing is uploaded to my media folder. And the browser console shows following error message:

```
jQuery.Deferred exception: $nImageInput.fileupload(...).done is not a function onImageUpload@http://192.168.56.3:8000/person/10043:365:30
de</t.prototype.triggerEvent@http://192.168.56.3:8000/static/summernote/summernote-bs4.min.js:3:114770
ne</t.prototype.show/<@http://192.168.56.3:8000/static/summernote/summernote-bs4.min.js:3:98707
e@http://192.168.56.3:8000/static/core/js/jquery-3.4.1.min.js:2:29453
l/</t<@http://192.168.56.3:8000/static/core/js/jquery-3.4.1.min.js:2:29755
```

Any suggestions how to further troubleshoot or resolve this issue would be greatly appreciated.

**Comment by Danussie** [comment]
I'm having the exact same problem, @tomas did you find a solution?

**Comment by tomas-nz** [comment]
@Danussie No, I don't have a solution unfortunately. I don't experience this particular issue if I use the iframe widget. But there I haven't found a way to call summernote function/API inside the iframe when my JavaScript code runs in the parent page. For example I would need to set and retrieve its content like this:

```
$(noteDetails).summernote('code', some_variable);
some_variable = $(noteDetails).summernote('code');
```

**Comment by tomas-nz** [comment]
@Danussie No, I don't have a solution unfortunately. I don't experience this particular issue if I use the iframe widget. But there I haven't found a way to call summernote function/API inside the iframe when my JavaScript code runs in the parent page. For example I would need to set and retrieve its content like this:

```
$(noteDetails).summernote('code', some_variable);
some_variable = $(noteDetails).summernote('code');
```

**Comment by wodn5515** [comment]
i had same problem when theme was set "lite". I changed it to "bs4" or "bs3" then it works.

**Comment by DeanElliott96** [comment]
Creating this file and including before the summernote JS file resolved this issue for me:
https://github.com/blueimp/jQuery-File-Upload/blob/master/js/jquery.fileupload.js

Related Stack Overflow: https://stackoverflow.com/questions/33943551/jquery-file-upload-typeerror-fileupload-is-not-a-function
